iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -614,6 +614,7 @@ public MemberExpression MEMBER_EXPRESSION_WITHOUT_DIRECT_EXPRESSION_RESOLVING()
             MEMBER_EXPRESSION_EXCLAMATION_COMPONENT(),
             MEMBER_EXPRESSION_FUNCTION_CALL_COMPONENT(),
             MEMBER_EXPRESSION_IDENTIFIER_COMPONENT(),
+            MEMBER_EXPRESSION_NESTED_RESOURCE_IDENTIFIER_COMPONENT(),
```

Comment:
For me, the name of this method is not very clear. Maybe rename the old `MEMBER_EXPRESSION_IDENTIFIER_COMPONENT` to `MEMBER_EXPRESSION_DOT_IDENTIFIER_COMPONENT` and `MEMBER_EXPRESSION_DOUBLE_COLON_IDENTIFIER_COMPONENT`?